podtato-server/kind/full_install.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+echo "Creating KIND cluster "
+
+kind create cluster --config=./cluster.yaml
```

Comment:
```suggestion
kind create cluster
```

The cluster configuration is missing.

Reply:
This will not work as we need a specific cluster config